MahApps.Metro/Styles/FlatToggleButton.xaml

```diff
@@ -0,0 +1,10 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
```

Comment:
@Koopakiller I think this files is not necessary, cause the `FlatButton.xaml` is obsolete (it is a residual waste)